Update _index.md (#23797)
DataDog/documentation: content/en/security/cloud_security_management/agentless_scanning/_index.md

2. For Lambda functions, the scanners fetch the function's code.
3. The scanner creates snapshots of EBS volumes used by EC2 instances. These snapshots serve as the basis for conducting scans. Using the snapshots, or the code, the scanner generates a list of packages.

> 4. After the scan is complete, only the list of packages is transmitted to Datadog, as well as information related to collected hosts (hostnames/EC2 instances), while all other data remains within your infrastructure. Snapshots created during the scan cycle are deleted.

Reviewer: ```suggestion
4. After the scan is complete, the list of packages and information related to collected hosts (hostnames/EC2 instances) are transmitted to Datadog, with all other data remaining within your infrastructure. Snapshots created during the scan cycle are deleted.
```
